v2/routes.go

```diff
@@ -67,26 +67,19 @@ func (s *Server) createSubscription(w http.ResponseWriter, r *http.Request) {
 		localmetrics.UpdateSubscriptionCount(localmetrics.FAILCREATE, 1)
 		return
 	}
-	if subExists, ok := s.pubSubAPI.HasSubscription(sub.GetResource()); ok {
+	clientIDs := s.subscriberAPI.GetClientIDByResource(sub.GetResource())
```

Comment:
why Plural clientID's?

Reply:
should be one clientID for the resource

Reply:
This returns `clientIDs []uuid.UUID`. I did not change the logic.
https://github.com/redhat-cne/sdk-go/blob/main/v1/subscriber/subscriber.go#L254